### PR TITLE
test: refactor and actualise Transactions tests

### PR DESCRIPTION
**Pull request by amelnytskyi**
# What ❔

Added coverage for all 0 items in responses for Transaction endpoints

## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.

### CODE REVIEW

File: packages/integration-tests/tests/api/transactions.test.ts [hunk 1]
```diff
@@ -323,6 +522,37 @@ describe("Transactions", () => {
         // expect(response.body).toStrictEqual(expect.objectContaining({ to: contract })) //unstable on CI
```
Comment (pcheremu) [inline packages/integration-tests/tests/api/transactions.test.ts:522]: It probably can be uncommented

Comment (pcheremu) [inline packages/integration-tests/tests/api/transactions.test.ts:522]:
```suggestion
        expect(response.body).toStrictEqual(expect.objectContaining({ to: contract }));
```

**Review by pcheremu — COMMENTED** [review]
Just commit my suggestion and I will be ready to approve.

File: packages/integration-tests/tests/api/transactions.test.ts (hunk 1 above)
Comment (amelnytskyi) [reply packages/integration-tests/tests/api/transactions.test.ts]: Requested change added

Comment (amelnytskyi) [reply packages/integration-tests/tests/api/transactions.test.ts]: Requested change added

**Review by pcheremu — APPROVED** [review]

**Pull request merged**